### DISCUSSION TITLE
Splunk configuration parameter Message_Key missing

### DISCUSSION THREAD

**Discussion by demian711** [fluent/fluentd#3709] [Q&A] [answered]
Hello, We recently noticed that in version 1.9 of fluent-bit, configuration parameter Message_Key is missing for Splunk.
Looking documentation, seems that it was deprecated in 1.8, but there is no reference at all for this deprecation.
Even more, its only noticed because it disappears from an example and that's it

<img width="958" alt="image" src="https://user-images.githubusercontent.com/4580564/161980927-5b8eae2c-f895-4dd4-b0a7-a473ca6635b2.png">

`[error] [config] splunk: unknown configuration property 'Message_Key'. The following properties are allowed: compress, http_user, http_passwd, http_buffer_size, http_debug_bad_request, event_key, event_host, event_source, event_sourcetype, event_sourcetype_key, event_index, event_index_key, event_field, splunk_token, splunk_send_raw, and channel.`

Is there any parameter to replace Message_key ?

many thanks

**Comment by cosmo0920** [comment, marked as answer]
Here is Fluentd community forum.
Could you post your question into https://github.com/fluent/fluent-bit/discussions instead?
Thanks!
